docs/tutorials/start-a-private-network/3keygen.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Generate Your Own Keys
+---
+
+Now that we know the fundamentals and the command line options, it's time to generate our own keys rather than using the well-known Alice and Bob keys. Each person who wants to participate in the blockchain should generate their own keys. This page explains several options for generating keys, and each participant only needs to choose one. Regardless of which option you choose, be sure to record all of the output from this section as you will need it later.
+
+## Option 1: Subkey
+Subkey is a tool the generates keys specifically designed to be used with Substrate. This is the most secure way to generate keys, and is how you should generate keys if you plan to use them in production.
```

Comment:
I think you should omit "this is the most secure... how you should generate keys".

We can create a separate page on securely generating keys if you would like to reference something like it, but I don't think this is advice we want to have on our docs.

The best and most secure way to generate keys is offline on a device which has never been on or will be on the internet.

```suggestion
Subkey is a tool the generates keys specifically designed to be used with Substrate.
```

Reply:
I think that the statement is correct but it could be more info than what is needed here.

Reply:
Thinking more about this statement. I think that is should be removed. The main reason is that even if it is true your console output is not encrypted and it is not the most secure way. It will depend on how the user stores the seed or private key.
In case of the UI there are also some attack vectors but at least the file that you get is encrypted and generated locally on client side.